sdk/entra/functions-authentication-events/review/functions-authentication-events.api.md

```diff
@@ -0,0 +1,144 @@
+## API Report File for "@azure/functions-authentication-events"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export interface ActionableCloudEventResponse<TEventAction extends AuthEventAction> extends ActionableResponse<TEventAction> {
+    // (undocumented)
+    oDataType: string;
+}
+
+// @public
+export interface ActionableResponse<TEventAction extends AuthEventAction> extends AuthEventResponse {
+    actions: TEventAction[];
+}
+
+// @public
+export interface AuthenticationEventContext {
+    authenticationProtocol: "OAUTH2.0" | "SAML" | "WS-FED" | "unknownFutureValue" | "";
+    client: AuthenticationEventContextClient;
+    clientServicePrincipal: AuthenticationEventContextServicePrincipal;
+    correlationId: string;
+    resourceServicePrincipal: AuthenticationEventContextServicePrincipal;
+    user: AuthenticationEventContextUser;
+}
+
+// @public
+export interface AuthenticationEventContextClient {
+    ip: string;
+    // (undocumented)
+    locale: string;
+    // (undocumented)
+    market: string;
+}
+
+// @public
+export interface AuthenticationEventContextServicePrincipal {
+    appDisplayName: string;
+    appId: string;
+    displayName: string;
+    id: string;
+}
+
+// @public
+export interface AuthenticationEventContextUser {
+    companyName: string;
+    country: string;
```

Comment:
not sure if this is something we can change, but if we can it would be nice to name it `contryCode`

Reply:
Unfortunately we have to match the C# library which matches the graph data contracts so we keep this the same for consistence.